pkg/providers/instancetype/types.go

```diff
@@ -228,7 +248,13 @@ func amdGPUs(info *ec2.InstanceTypeInfo) *resource.Quantity {
 
 func awsNeurons(info *ec2.InstanceTypeInfo) *resource.Quantity {
 	count := int64(0)
-	if info.InferenceAcceleratorInfo != nil {
+	if *info.InstanceType == "trn1.2xlarge" {
```

Comment:
can you add a TODO comment here as well so we remember to remove these hardcoded values once the values are published in an API?